benchmarks/flowertune-llm-leaderboard/README.md

```diff
@@ -0,0 +1,87 @@
+![](_static/flower_llm.jpg)
+
+# FlowerTune LLM Leaderboard
+
+This repository guides you through the process of federated LLM instruction tuning with pre-trained [LLama2](https://huggingface.co/mistralai/Mistral-7B-v0.3) models across various tasks.
+Please follow the instructions to run baseline tasks and create your own projects.
+
+
+## Structure
+Each LLM task within this directory is completely self-contained, with its source code neatly organized in its respective subdirectory.
+The task subdirectories contain the following structure:
+
+```bash
+llm-flowertune/<task-name>/
+                 ├── README.md
+                 ├── main.py         # <- Flower ClientApp/ServerApp build
+                 ├── client.py       # <- Flower client constructor
+                 ├── server.py       # <- Sever-related functions
+                 ├── model.py        # <- Model build
+                 ├── dataset.py      # <- Dataset and tokenizer build
+                 ├── requirements.txt         # <- Environment dependencies
+                 ├── conf/config.yaml         # <- User configuration
+                 ├── static_conf/config.yaml  # <- Static configuration
+                 └── <evaluation>             # <- Evaluation metrics
+```
+> More tasks will come soon.
+
+## Running FlowerTune-LLM tasks
+
+Each LLM-FlowerTune task is self-contained in its own directory.
+Running a task can be done by:
+
+1. Follow the `Environments setup` section in the `README.md` of each task subdirectory to create a new project directory.
+
+
+2. Navigate inside the directory of the task you'd like to run.
+
+
+3. Install project dependencies are defined in `requirements.txt`. We recommend [Conda](https://conda.io/projects/conda/en/latest/user-guide/getting-started.html) to install those dependencies and manage your virtual environment, but feel free to use a different way of installing dependencies and managing virtual environments if you have other preferences.
+
+    ```shell
+    conda create -n llm-flowertune python==3.10.0
+    conda activate llm-flowertune
+    pip install -r requirements.txt
+    ```
```

Comment:
This looks like too much detail for the top-level directory. Should we better keep it as:

```
3. Install project dependencies are defined in `requirements.txt`.
```